discord/ui/view.py

```diff
@@ -252,7 +252,7 @@ def from_message(cls, message: Message, /, *, timeout: Optional[float] = 180.0)
             view.add_item(_component_to_item(component))
         return view
 
-    def add_item(self, item: Item[Any]) -> None:
+    def add_item(self, item: Item[Any]) -> View:
```

Comment:
```suggestion
    def add_item(self, item: Item[Any]) -> Self:
```
We use `typing.Self` to denote when the self-type is being returned.